mbuild/tests/test_lammpsdata.py

```diff
@@ -17,6 +18,17 @@ def test_save_box(self, ethane):
         box = mb.Box(lengths=np.array([2.0, 2.0, 2.0]))
         ethane.save(filename='ethane-box.lammps', forcefield_name='oplsaa', box=box)
 
+    def test_nbfix(self, ethane):
+        from foyer import Forcefield
+
+        OPLSAA = Forcefield(name='oplsaa')
+        structure = OPLSAA.apply(ethane)
+        # Add nbfixes
+        types = list(set([a.atom_type for a in structure.atoms]))
+        types[0].add_nbfix(types[1].name, 1.2, 2.1)
+        types[1].add_nbfix(types[0].name, 1.2, 2.1)
+        write_lammpsdata(filename='nbfix.lammps', structure=structure)
```

Comment:
The tests for the LAMMPS writer in general need a major facelift, which is for another PR. But to at least start to chip away at improving these, can you also provide a check here that the file that's written actually contains the expected Pair IJ section and that the output is correct?